freedomofpress/securethenews
# Converts dev env to use Docker

Ready for review.

Closes #83.

Leverages the new Molecule-based testing and development workflow to spin up Docker containers for local development, rather than full VMs. Should make for a much more pleasant experience locally.

To test:

1. Confirm CI is passing (both Travis and Circle).
2. Run `make dev-go` and wait for the service to come up.
3. Browse to the localhost port displayed on screen and navigate through the site.
4. Confirm you can use the site locally and no errors are reported.
